src/CSVReader.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Created by ken-linux2 on 14/07/16.
+ */
+import com.sun.xml.internal.ws.*;
+
+import java.io.*;
+import java.io.Closeable;
+
+
+public class CSVReader{
+    public static void main(String[] args) {
+        CSVReader RCF = new CSVReader();
+        RCF.run();
+    }
+
+    private void run() {
+        final String CSV_File = "/home/ken-linux2/Downloads/macycExampleFile.csv - macycExampleFile.csv.csv";
+        BufferedReader br = null;
+        String line;
+        final String SAPERATOR = ",";
+        String[] myFile;
+        Account account=new Account();
+        Product product=new Product();
+        Channel channel=new Channel();
+        AdGroup adGroup=new AdGroup();
+
+        try {
+            br = new BufferedReader(new FileReader(CSV_File));
+            br.readLine();
+            while ((line = br.readLine()) != null) {
+                myFile = line.split(SAPERATOR,17);
+                account.setName(myFile[6]);
```

Comment:
Looks good. Please move this logic to a Converter class that receives a String (line from the file) and return a Channel and add a unit test to the converter and to the reader.